### DISCUSSION TITLE
Is Prisma Client HIPAA Compliant?

### DISCUSSION THREAD

**Discussion by rronalddas** [prisma/prisma#6720] [Q&A] [answered]
Hi, I am planning to use Prisma for a new project, that will deal with Healthcare data. Since Prisma Client uses a Rust Binary internally, am wondering if it is HIPAA compliant?

**Comment by janpio** [comment, marked as answer]
Hi @rronalddas, I can unfortunately not give you advise on if Prisma Client is HIPAA compliant or not - I just do not know enough about that and we did not have this question before.

What I can tell you is that the Rust binary is downloaded right next to the Client library, and runs next to it on your deployment or computer when you use Prisma Client. That binary then talks to your database server as and database client library also does. There are no Prisma servers, or anything external involved.

**Reply by rronalddas** [reply]
@janpio Thanks for the quick reply, this was actually a blocker, since i wasn't sure if the rust binary collected any data for analytics purpose to be used by Prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
